File: LR_3-4/Pack2.py
# Задача 2.6

cnt = int(input("Кол-во учеников: "))
sum_res = 0

for _ in range(cnt):
    sum_res += int(input(f"Введите рост {cnt + 1}: "))

print(round(sum_res / cnt * 100) / 100)

File: LR/LR_3-4/Pack2.pas
{ Задача 2.6 }

var
  cnt, i, value: integer;
  sum_res: real;
begin
  sum_res := 0;
  writeln('Кол-во учеников: ');
  readln(cnt);

  for i := 1 to cnt do
  begin
    writeln('Рост: ');
    readln(value);
    sum_res := sum_res + value;
  end;

  writeln(Round((sum_res / cnt) * 100) / 100);
end.